src/com_tjnotifications/admin/sql/install.mysql.utf8.sql

```diff
@@ -26,9 +26,9 @@ CREATE TABLE IF NOT EXISTS `#__tj_notification_template_configs` (
   `template_id` int(11) NOT NULL DEFAULT 0,
   `backend` varchar(50) NOT NULL DEFAULT '',
   `language` char(7) NOT NULL DEFAULT '*',
-  `subject` text NOT NULL DEFAULT '',
-  `body` text NOT NULL DEFAULT '',
-  `params` text NOT NULL DEFAULT '',
+  `subject` text NOT NULL,
+  `body` text NOT NULL,
+  `params` text NOT NULL,
```

Comment:
params could be null

`params` text NOT NULL DEFAULT '' =>`params` text  DEFAULT NULL

makes more sense